airbyte-integrations/connectors/source-stripe/README.md

```diff
@@ -61,17 +61,26 @@ To run unit tests locally, from the connector directory run:
 python -m pytest unit_tests
 ```
 
+#### Acceptance Tests
+Customize `acceptance-test-config.yml` file to configure tests. See [Source Acceptance Tests](source-acceptance-tests.md) for more information.
+If your connector requires to create or destroy resources for use during acceptance tests create fixtures for it and place them inside integration_tests/acceptance.py.
+To run your integration tests with acceptance tests, from the connector root, run
+```
+python -m pytest integration_tests -p integration_tests.acceptance
+```
+To run your integration tests with docker
+
 ### Locally running the connector docker image
 
 #### Build
 First, make sure you build the latest Docker image:
 ```
-docker build . -t airbyte/source-stripe:dev
+docker build --no-cache . -t airbyte/source-stripe:dev
```

Comment:
why `--no-cache`?

Reply:
locally, if you rebuild the docker image multiple times in a row it saves the cache and doesn't accept the minor changes, let's say changes in the schemas. So put this as it is, and it solved the issue with the docker image.